cinema/migrations/0002_initial.py

```diff
@@ -18,17 +18,23 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name="order",
             name="user",
-            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL),
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL
```

Comment:
Ensure that the `AUTH_USER_MODEL` is correctly set up in your Django settings and that the `Order` model's `user` field correctly references it.